### ISSUE TITLE
Can't resolve 'react-font-size-changer'

### ISSUE THREAD

**Issue by abdu355** [mhmtztmr/react-font-size-changer#1]
When importing the lib: ` import { FontSizeChanger } from "react-font-size-changer";`
I get this error:  **_"Can't resolve 'react-font-size-changer'"_**

Installed using 
`"yarn add react-font-size-changer"`

package.json content:  
```
"react": "^16.4.2",
"react-dom": "^16.4.2",
"react-font-size-changer": "^0.0.2",
```

If I add react-font-size-changer/dist/bundle it works, but I get the following error when clicking the buttons: 

_**this.updater.enqueuecallback is not a function**_

**Comment by savannahostrowski** [comment]
@abdu355 Did you end up figuring this out? I'm encountering the same issue!

**Comment by abdu355** [comment]
Hey @savannahostrowski ,
Unfortunately no. Ended up creating a button that modifies my <p> tag font size.

**Comment by savannahostrowski** [comment]
@abdu355 Same here.

**Comment by mhmtztmr** [comment]
Hey @abdu355 @savannahostrowski
Thanks for raising the issue.
Appending dist/bundle is required to make it work since "main":"dist/bundle.js" is missing. I will be adding it.
For the other issue "this.updater.enqueuecallback is not a function", it is due to React version incompatibility since the component is built with React 15. I will be updating React version from 15 to 16.

Thanks again.

**Comment by mhmtztmr** [comment]
Hey,

The fix is provided as of v0.1.1
